core/src/main/java/nl/nn/adapterframework/scheduler/job/SendMessageJob.java

```diff
@@ -55,19 +57,16 @@ public void configure() throws ConfigurationException {
 
 	@Override
 	public void execute() throws JobExecutionException, TimeoutException {
-		try {
-			localSender.open();
-			//sendMessage message cannot be NULL
-			Message message = new Message((getMessage()==null) ? "" : getMessage());
-			PipeLineSession session = new PipeLineSession();
+		try (Message toSendMessage = new Message((getMessage() == null) ? "" : getMessage());
+				PipeLineSession session = new PipeLineSession()) {
 			//Set a messageId that will be forwarded by the localSender to the called adapter. Adapter and job will then share a Ladybug report.
 			session.put(PipeLineSession.CORRELATION_ID_KEY, UUIDUtil.createSimpleUUID());
-			localSender.sendMessageOrThrow(message, session);
-		}
-		catch (SenderException e) {
-			throw new JobExecutionException("unable to send message to javaListener ["+javaListener+"]", e);
-		}
-		finally {
+
+			localSender.open();
+			localSender.sendMessageOrThrow(toSendMessage, session).close();
```

Comment:
Ik weet niet of het resultaat van `sendMessageOrThrow` een `null` zou kunnen zijn?

Reply:
Als het senden is mislukt, gooit hij een sendMessageOrThrow. In de andere gevallen is hij gevuld.